models/users.py

```diff
@@ -7,4 +7,5 @@ def __init__(self, id, email, name, password):
         self.id = id
         self.email = email
         self.name = name
-        self.password = password
\ No newline at end of file
+        self.password = password
+        self.valid = True
```

Comment:
Instead of adding a 'true' value to a user, and sending the whole user back, What if instead you CHECKED to see that you got a user back from your sql query, and then returned a dictionary with the id, a property of 'valid' on it. I don't think your users should get a new property, instead you should send a response dict back that simply has the info you need.